@planetarium/account-web3-secret-storage/src/Web3KeyStore.ts

```diff
@@ -0,0 +1,218 @@
+import { KeyId } from "./KeyId.js";
+import { PassphraseEntry } from "./PassphraseEntry.js";
+import {
+  type AccountDeletion,
+  type AccountGeneration,
+  type AccountImportation,
+  type AccountMetadata,
+  type AccountRetrieval,
+  Address,
+  type ImportableKeyStore,
+  RawPrivateKey,
+} from "@planetarium/account";
+import {
+  type KeystoreAccount,
+  decryptKeystoreJson,
+  encryptKeystoreJson,
+  isKeystoreJson,
+} from "ethers";
+import { Dirent } from "node:fs";
+import * as fs from "node:fs/promises";
+import { homedir } from "node:os";
+import * as path from "node:path";
+
+export interface Web3KeyStoreOptions {
+  path?: string;
+  passphraseEntry: PassphraseEntry;
+}
+
+/**
+ * Determines the default key store path.  It depends on the platform:
+ *
+ * - Linux/macOS: `$HOME/.config/planetarium/keystore`
```

Comment:
As a conclusion of our discussion (@Akamig, @dahlia, and @tkiapril,) we tentatively agreed that; `Application Support` is a variable directory where the files might be moved if the applications requiring the directory is not present. However, a key being removed without user's explicit action may have adverse effect, so it might be better to have a directory that is not intended for automated deletion without user's discretion to be used in the macOS launcher.